build/NuSpecs/UmbracoCms.SqlCe.nuspec

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata minClientVersion="4.1.0">
+        <id>UmbracoCms.SqlCe</id>
+        <version>9.0.0</version>
+        <title>Umbraco Cms Core Binaries</title>
+        <authors>Umbraco HQ</authors>
+        <owners>Umbraco HQ</owners>
+        <license type="expression">MIT</license>
+        <projectUrl>https://umbraco.com/</projectUrl>
+        <iconUrl>https://umbraco.com/dist/nuget/logo-small.png</iconUrl>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>Contains the SQL CE assemblies needed to run Umbraco Cms. This package only contains assemblies and can be used for package development. Use the UmbracoCms package to setup Umbraco in Visual Studio as an ASP.NET Core project.</description>
+        <summary>Contains the SQL CE assemblies needed to run Umbraco Cms</summary>
+        <language>en-US</language>
+        <tags>umbraco</tags>
+        <repository type="git" url="https://github.com/umbraco/umbraco-cms" />
+        <dependencies>
+
+            <group targetFramework="net472">
+                <!--
+                  note: dependencies are specified as [x.y.z,x.999999) eg [2.1.0,2.999999) and NOT [2.1.0,3.0.0) because
+                  the latter would pick anything below 3.0.0 and that includes prereleases such as 3.0.0-alpha, and we do
+                  not want this to happen as the alpha of the next major is, really, the next major already.
+                -->
+                <dependency id="UmbracoCms.Core" version="[$version$]" />
```

Comment:
I guess this reference has to be netstandard, as `UmbracoCms.Core` will be netstandard 2.0

Reply:
You are right! Will refactor, thanks 👍